docs/how-to-use-the-playground.md

```diff
@@ -24,44 +25,108 @@ Install Git (optional), Docker, Docker Compose.
 The playground runs several services. The TCP ports used may clash with existing services you run, such as MySQL or Postgres.
 
 | Docker container      | Ports used             |
-|-----------------------|------------------------|
+| --------------------- | ---------------------- |
 | playground-gravitino  | 8090 9001              |
 | playground-hive       | 3307 19000 19083 60070 |
 | playground-mysql      | 13306                  |
 | playground-postgresql | 15342                  |
 | playground-trino      | 18080                  |
 | playground-jupyter    | 18888                  |
+| playground-prometheus | 19090                  |
+| playground-grafana    | 13000                  |
 
 ## Playground usage
 
-### Launch playground
 
-You can launch the playground in two ways:
-
-1. Use a single curl command
 
+### One curl command launch playground
 ```shell
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/apache/gravitino-playground/HEAD/install.sh)"
 ```
 
-2. Use Git
+### Use git to download and launch playground
 
 ```shell
 git clone git@github.com:apache/gravitino-playground.git
 cd gravitino-playground
-./playground.sh start
 ```
 
-### Check status
+#### Docker
+
+##### Start
+
+```
+./playground.sh docker start
+```
+
+##### Check status
 ```shell 
-./playground.sh status
+./playground.sh docker status
+```
+##### Stop playground
+```shell
```

Comment:
Also please add a blank line above. Typically we should add a blank line between code blocks or paragraphs.